api/api/events/test_list_create_events.py

```diff
@@ -0,0 +1,173 @@
+    # given
+    test_user.is_staff = True
+    client.force_authenticate(test_user)
+
+    # when
+    response = client.post('/events/', {'name': 'Introduction to Git and Github',
+                                        'description': 'some desc',
+                                        'starts_at': '2022-08-07',
+                                        'ends_at': '2022-08-07',
+                                        'image_url': 'https://asite.com',
+                                        'address': 'c',
+                                        'club': test_club.id
+                                        })
+
+    # then
+    created_event = Events.objects.get(id=response.data['id'])
+
+    assert response.status_code == status.HTTP_201_CREATED
+    assert response.data == {'id': created_event.id,
+                             'name': 'Introduction to Git and Github',
+                             'description': 'some desc',
+                             'starts_at': '2022-08-07',
+                             'ends_at': '2022-08-07',
+                             'registration_required': False,
+                             'created_by': None,
+                             'image_url': 'https://asite.com',
+                             'club': 1,
+                             'registration_approval_required': False,
+                             'address': 'c'}
+    assert created_event.name == 'Introduction to Git and Github'
+
+
+@pytest.mark.django_db
+def testCreateEvents_unauthorizedUserRequest_throwsForbidden(client, test_user, test_club):
+    # given
+    client.force_authenticate(test_user)
+
+    # when
+    response = client.post('/events/', {'name': 'Introduction to Git and Github',
+                                        'description': 'some desc',
+                                        'starts_at': '2022-08-07',
+                                        'ends_at': '2022-08-07',
+                                        'image_url': 'https://asite.com',
+                                        'address': 'c',
+                                        'club': test_club.id
+                                        })
+    # then
+    assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+@pytest.mark.django_db
+def testCreateEvents_missingDataInRequest_throwsBasRequest(client, test_user, test_club):
+    # given
+    client.force_authenticate(test_user)
+
+    # when
+    # request missing start and end time
+    response = client.post('/events/', {'name': 'Introduction to Git and Github',
+                                        'description': 'some desc',
+                                        'image_url': 'https://asite.com',
+                                        'address': 'c',
+                                        'club': test_club.id
+                                        })
+
+    # then
+    assert response.status_code == status.HTTP_400_BAD_REQUEST
```

Comment:
Should we also test, what the error message is? It should clearly state what's missing.